brainscore_vision/models/cornet_s/__init__.py

```diff
@@ -0,0 +1,8 @@
+from brainscore_vision import model_registry
+from .helpers.helpers import CORnetCommitment, _build_time_mappings
+from .model import get_model, get_layers, TIME_MAPPINGS
+
+
+model_registry['cornet_s'] = lambda: CORnetCommitment(identifier='cornet_s', activations_model=get_model('cornet_s'),
```

Comment:
shouldn't this be `CORnet-S` for the registry and the identifier? That's how it is displayed on the website

Reply:
Updated the identifier casing.